contributing_to_docs/doc_guidelines.adoc

```diff
@@ -122,6 +122,31 @@ To use a discrete heading, just add `[discrete]` to the line before your unique
 == Managing authorization policies
 ----
 
+=== Creating unique IDs for 'Additional resources' and 'Next steps' titles
+
+Use unique IDs for *Additional resources* and *Next steps* titles. You can append `-additional-resources` or `-next-steps` to a unique string that describes the assembly topic.
+
+The following examples include IDs that are unique to the *Understanding the monitoring stack* assembly.
+
+*Example unique ID for an 'Additional resources' title*
+
+----
+[id="understanding-the-monitoring-stack-additional-resources"]
+== Additional resources
+----
+
+*Example unique ID for a 'Next steps' title*
+
+----
+[id="understanding-the-monitoring-stack-next-steps"]
+== Next steps
+----
+
+[NOTE]
+====
+Do not include IDs for *Additional resources* or *Next steps* subsections that use dot titles, such as `.Additional resources`.
+====
```

Comment:
I'll share here in case someone else wants to "second" it.  I believe asciidoctor refers to these as block titles.  If no one else cares, so be it.

Reply:
@mikemckiernan thank you for highlighting this. This is now resolved.